lib/tlscontext.h

```diff
@@ -44,6 +44,16 @@ typedef enum
   TVM_REQUIRED=0x0020,
 } TLSVerifyMode;
 
+typedef enum
+{
+  TSO_NONE,
+  TSO_NOSSLv2=0x0001,
+  TSO_NOSSLv3=0x0002,
+  TSO_NOTLSv1=0x0004,
+  TSO_NOTLSv11=0x0008,
+  TSO_NOTLSv12=0x0010,
+} TLSSslOptions;
+
```

Comment:
hi,

I'm reluctant to merge the patch in this form.
Please, use `int` as return type, as I suggested before (and you can keep `enum`... ).

L.